build/root/WORKSPACE

```diff
@@ -70,7 +70,7 @@ docker_pull(
     digest = "sha256:d83594ecd85345144584523e7fa5388467edf5d2dfa30d0a1bcbf184cddf4a7b",
```

Comment:
please update this to `sha256:cc782ed16599000ca4c85d47ec6264753747ae1e77520894dca84b104a7621e2`